docs/naming_conventions.md

```diff
@@ -1,36 +1,36 @@
+# Global character set and size limitations
+Globally, the platform impose certain rules when defining the data model. These rules directly come from Orion Context Broker [constraints](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#general-syntax-restrictions) regarding the character set used for entity and attribute names and their types:
 
 * The following charaters are forbidden: `<`, `>`, `"`, `'`, `=`, `;`, `(` and `)`.
 * The following ones should also be avoided: white space, `?`, `/`, `%` and `&`.
 
 Additionally, Orion Context Broker imposes the following restrictions:
 
-* The [FIWARE-service](http://fiware-orion.readthedocs.io/en/master/user/multitenancy/index.html) must be a string of alphanumeric characters including the underscore, whose maximum length is 50 characters.
-* The [FIWARE-servicePath](http://fiware-orion.readthedocs.io/en/master/user/service_path/index.html) must be a string defining a Unix-like path starting with slash and having maximum 10 levels; being each level a string of alphanumeric characters including the underscore, whose maximum lenght is 50 characters. Nevertheless, the platform currently only allows a single level.
+* The [FIWARE-service](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#multi-tenancy) must be a string of alphanumeric characters including the underscore, whose maximum length is 50 characters.
+* The [FIWARE-servicePath](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#service-path) must be a string defining a Unix-like path starting with slash and having maximum 10 levels; being each level a string of alphanumeric characters including the underscore, whose maximum lenght is 50 characters. Nevertheless, the platform currently only allows a single level.
 
-All these limitations imposed by Orion Context Broker are propagated to the storages via Cygnus connectors with MySQL and MongoDB.
+All these limitations imposed by Orion Context Broker are propagated to the storages via Cygnus connectors with PostgreSQL and MongoDB.
 
-#MySQL character set and size limitations
-Persisting context data in [MySQL](http://fiware-cygnus.readthedocs.io/en/master/cygnus-ngsi/flume_extensions_catalogue/ngsi_mysql_sink/index.html) implies creating databases and tables, whose names are constrained to 64 character strings of alphanumerics and underscore:
+# PostgreSQL character set and size limitations
+Persisting context data in [PostgreSQL](https://fiware-cygnus.readthedocs.io/en/master/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.html) implies creating databases and tables, whose names are constrained to 63 character strings of alphanumerics and underscore:
```

Comment:
Why almost all links to readthedocs were changed to the corresponding Github repo but this is still using readthedocs?